Vagrantfile

```diff
@@ -80,11 +84,12 @@ Vagrant.configure('2') do |config|
   config.vm.provider :virtualbox do |vb|
     vb.customize ['modifyvm', :id, '--natdnsproxy1', 'off']
     vb.customize ['modifyvm', :id, '--natdnshostresolver1', 'off']
-    vb.customize ['modifyvm', :id, '--memory', 1152]
+    vb.customize ['modifyvm', :id, '--memory', 1536]
   end
 
   config.vm.define :chef do |cfg|
-    cfg.vm.provision :shell, inline: 'rpm -q chefdk || curl -L https://omnitruck.chef.io/install.sh | bash -s -- -P chefdk -v 4.13.3'
+    # update VERSION to take newer packages (current version is the lastest that still has chef-client 16) https://docs.chef.io/release_notes_workstation/
+    cfg.vm.provision :shell, inline: 'VERSION="21.4.365" && yum install -y https://packages.chef.io/files/stable/chef-workstation/${VERSION}/el/8/chef-workstation-${VERSION}-1.el7.x86_64.rpm'
```

Comment:
oddly, this seems to work, but shouldn't this be `...${VERSION}-1.el8.x86_64.rpm` (rather than -1.el7.x86)?

Reply:
Yeah I thought that would be the case too, but it doesn't exist with that path:
```
04:50 PM # VERSION="21.4.365" && wget --spider https://packages.chef.io/files/stable/chef-workstation/${VERSION}/el/8/chef-workstation-${VERSION}-1.el7.x86_64.rpm

Spider mode enabled. Check if remote file exists.
--2023-01-26 16:50:24--  https://packages.chef.io/files/stable/chef-workstation/21.4.365/el/8/chef-workstation-21.4.365-1.el7.x86_64.rpm
Resolving packages.chef.io (packages.chef.io)... 199.232.10.110
Connecting to packages.chef.io (packages.chef.io)|199.232.10.110|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 176038822 (168M) [application/x-rpm]
Remote file exists.

04:50 PM # VERSION="21.4.365" && wget --spider https://packages.chef.io/files/stable/chef-workstation/${VERSION}/el/8/chef-workstation-${VERSION}-1.el8.x86_64.rpm

Spider mode enabled. Check if remote file exists.
--2023-01-26 16:50:30--  https://packages.chef.io/files/stable/chef-workstation/21.4.365/el/8/chef-workstation-21.4.365-1.el8.x86_64.rpm
Resolving packages.chef.io (packages.chef.io)... 199.232.10.110
Connecting to packages.chef.io (packages.chef.io)|199.232.10.110|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
Remote file does not exist -- broken link!!!
```

Reply:
Yeah there are only el7 named files in that dir:
https://packages.chef.io/files/stable/chef-workstation/21.4.365/el/8/